user-guide/rate-limiting-tutorial.md

```diff
@@ -28,7 +28,7 @@ metadata:
 spec:
   service: "example-rate-limit:5000"
 ---
-apiVersion: v1
+apiVersion: v2
```

Comment:
Sorry should have been clearer. This should remain `v1` as it is a kubernetes api